Models/getModels.js

```diff
@@ -16,13 +17,61 @@ exports.selectCategories = () => {
     });
 };
 
-exports.selectReviews = () => {
-  return db
-    .query(
-      `
-      SELECT *, COUNT(review_id) AS comment_count FROM reviews group by review_id;
-  `,
-    )
+exports.selectReviews = (categories, sortBy, order, query) => {
```

Comment:
Look at setting parameters to default values when they aren't provided. It will make this much easier 👍